docs/community/pr-velocity.md

```diff
@@ -0,0 +1,11 @@
+---
+id: pr-velocity
+title: 'PR Velocity'
+sidebar_label: 'PR Velocity'
+keywords:
+  - pr-velocity 
+---
+
+The PR Velocity as shown in purple in the screenshot below, measures the percentage and amount of days that  pull requests that are merged. It's a great way to track the progress of your open source contributions. Think of it as keeping track of the pizza that is baking in your oven.
```

Comment:
```suggestion
The PR Velocity as shown in purple in the screenshot below, measures the percentage and amount of days that  pull requests that are merged. It's a great way to track the progress of your open source contributions.
```

Remove the the extra context. I think that is better suited for a blog post. Just considering that would not translate well in another language.